abs.yaml

```
@@ -0,0 +1,2 @@
+channels:
+  - https://staging.continuum.io/prefect/fs/geos-feedstock/pr11/b86b987
```

Can you build it for py313 too?
```suggestion
  - https://staging.continuum.io/prefect/fs/geos-feedstock/pr11/b86b987
  
  build_env_vars:
  ANACONDA_ROCKET_ENABLE_PY313 : yes
  
```

---

I was just going to add this, thanks!